site2/docs/client-libraries-cpp.md

```diff
@@ -14,40 +14,100 @@ Pulsar C++ client is supported on **Linux** and **MacOS** platforms.
 
 [Doxygen](http://www.doxygen.nl/)-generated API docs for the C++ client are available [here](/api/cpp).
 
+## System requirements
+
+You need to have the following installed to use the C++ client:
+
+* [CMake](https://cmake.org/)
+* [Boost](http://www.boost.org/)
+* [Protocol Buffers](https://developers.google.com/protocol-buffers/) 2.6
+* [Log4CXX](https://logging.apache.org/log4cxx)
+* [libcurl](https://curl.haxx.se/libcurl/)
+* [Google Test](https://github.com/google/googletest)
+* [JsonCpp](https://github.com/open-source-parsers/jsoncpp)
+
 ## Linux
 
+### Compilation 
```

Comment:
I think it is better to use an ordered list to refine this part. Ans use the period (.) at the end of each sentence instead of a colon (:). Please check through the whole updates.